_data/language.yaml

```diff
@@ -23,3 +23,7 @@ array:
   - name: C++
     url: //github.com/opentracing/opentracing-cpp
     icon: ''
+
+  - name: Contributions
+    url: //github.com/opentracing-contrib
```

Comment:
@pavolloffay , also, you should change the url to `https://github.com/opentracing-contrib/meta`, as Ben mentioned before.